src/stories/containers/cu-table/cu-table.tsx

```diff
@@ -373,8 +373,21 @@ export const CuTable = () => {
       <SEOHead
         title="MakerDAO Ecosystem Performance Dashboard | Maker Expenses"
         description="MakerDAO Ecosystem Performance Dashboard provides a transparent analysis of Core Unit teams' finances, projects, and their position in the DAO."
-        image="/favicon-192.png"
-      />
+        image="https://expenses-dev.makerdao.network/favicon.png"
```

Comment:
`favicon.png` image does not exist anymore, try to use the new ones. It could be a good idea to use relative paths instead of an absolute path pointing to an environment that could be down when the actual production environment is not

Reply:
done